command/format.go

```diff
@@ -125,6 +126,28 @@ func (j JsonFormatter) Output(ui cli.Ui, secret *api.Secret, data interface{}) e
 	return nil
 }
 
+// An output formatter for raw output of the original request object
+type RawFormatter struct{}
+
+func (r RawFormatter) Format(data interface{}) ([]byte, error) {
+	byte_data, ok := data.([]byte)
+	if !ok {
+		panic("backtrace")
```

Comment:
I'm guessing we don't want to leave this panic in?

Reply:
Ah, that's fair. Clarified the error message too.